deploy.sh

```diff
@@ -1,13 +1,21 @@
 #!/bin/bash -x
 
-if [[ -z "$MBTA_V2_API_KEY" ]]; then
-    echo "Must provide MBTA_V2_API_KEY in environment" 1>&2
+if [[ -z "$MBTA_V2_API_KEY" || -z "$DD_API_KEY"  ]]; then
```

Comment:
Do I need a DD_API_KEY to deploy locally now? Are there any docs/instructions we need to update?

Reply:
Only for deployment. Probably need to update docs. You can use any DD_API_KEY from our account, anyone with the datadog account can get it easily in the UI